src/lib.rs

```diff
@@ -72,12 +72,29 @@ pub async fn with_actor_system<F: FnOnce(&riker::actors::ActorSystem)>(cb: F) {
     cb(&runtime.stronghold.system)
 }
 
-/// Opens the IOTA app on Ledger (Nano S/X or Speculos simulator).
+/// The ledger status.
```

Comment:
```suggestion
/// The Ledger device status.
```

Just to clarify that we're talking about the hardware wallet, and not the ledger on the Tangle

Reply:
nice catch, thanks